docs/source/conf.py

```diff
@@ -0,0 +1,217 @@
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This patterns also effect to html_static_path and html_extra_path
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'lib', 'lib64','**/_common/*', 'README.md', '.git', '.github', '_utils', '_templates', 'rst_include']
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = True
+
+# Setup Sphinx
+def setup(sphinx):
+    sphinx.add_config_value('recommonmark_config', {
+        'enable_eval_rst': True,
+        'enable_auto_toc_tree': False,
+    }, True)
+    sphinx.add_transform(AutoStructify)
+
+# Adds version variables for monitoring and manager versions when used in inline text
+rst_prolog = """
+.. |mon_version| replace:: 3.1
+.. |man_version| replace:: 2.0
+.. |mon_root| replace::  :doc:`Scylla Monitoring Stack </operating-scylla/monitoring/index>`
+""" 
+
+# -- Options for HTML output ----------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'sphinx_scylladb_theme'
+html_theme_path = ["../.."]
+html_style = ''
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+
+html_theme_options = {
+    'header_links': [
+    ('Scylla Rust Driver', 'https://scylladb.github.io/rust-driver/'),
+    ('Scylla Cloud', 'https://docs.scylladb.com/scylla-cloud/'),
+    ('Scylla University', 'https://university.scylladb.com/'),
+    ('ScyllaDB Home', 'https://www.scylladb.com/')],
+    'site_description': 'Scylla Driver for Rust.',
+    'github_issues_repository': 'scylladb/scylla-rust-driver',
+    'show_sidebar_index': 'false',
+    'hide_banner': 'true',
+    'default_branch': 'stable'
+}
+
+extlinks = {
+    'manager': ('/operating-scylla/manager/%s/',''),
+    'manager_lst': ('/operating-scylla/manager/2.0/%s/',''),
+    'monitor': ('/operating-scylla/monitoring/%s/',''),
+    'monitor_lst': ('/operating-scylla/monitoring/3.1/%s/','')
+}
+
+# If not None, a 'Last updated on:' timestamp is inserted at every page
+# bottom, using the given strftime format.
+# The empty string is equivalent to '%b %d, %Y'.
+#
+html_last_updated_fmt = '%d %B %Y'
+
+# Custom sidebar templates, maps document names to template names.
+#
+html_sidebars = {'**': ['side-nav.html']}
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'ScyllaDocumentationdoc'
+
+# URL which points to the root of the HTML documentation. 
+html_baseurl = 'https://sphinx-theme.scylladb.com'
```

Comment:
Replace by http://rust-driver.docs.scylladb.com/. Remember that the owner of the scylladb.com domain should point a CNAME record from rust-driver.docs.scylladb.com to scylladb.github.io. More info at https://sphinx-theme.scylladb.com/stable/github-pages#setting-up-a-custom-domain

Reply:
@dgarcia360 we can do the CNAME change once we finish here